packages/components-css/button-css/src/html/_mixin.scss

```diff
@@ -1,7 +1,69 @@
 @use "../mixin";
 
-@mixin button {
+@mixin nl-button--html {
+  input[type="button" i],
+  input[type="reset" i],
+  input[type="submit" i],
   button {
+    @include mixin.nl-button--default;
     @include mixin.nl-button;
   }
+
+  input[type="button" i][hidden],
+  input[type="reset" i][hidden],
+  input[type="submit" i][hidden],
+  button[hidden] {
+    display: none;
+  }
+
+  input[type="button" i]:where([disabled]),
+  input[type="reset" i]:where([disabled]),
+  input[type="submit" i]:where([disabled]),
+  button:where([disabled]),
+  input[type="button" i]:where([aria-disabled="true"]),
+  input[type="reset" i]:where([aria-disabled="true"]),
+  input[type="submit" i]:where([aria-disabled="true"]),
+  button:where([aria-disabled="true"]) {
+    @include mixin.nl-button--disabled;
+    @include mixin.nl-button--default-disabled;
+  }
+
+  button[type="submit" i],
+  button[aria-disabled="true"][type="submit" i],
+  button:disabled[type="submit" i],
+  input[type="submit" i],
+  input[aria-disabled="true"][type="submit" i],
+  input:disabled[type="submit" i] {
+    // Submit cursor has precedence over the disabled cursor.
+    @include mixin.nl-button--submit;
+  }
+
+  input[type="button" i]:where([aria-pressed="true"]),
+  input[type="reset" i]:where([aria-pressed="true"]),
+  input[type="submit" i]:where([aria-pressed="true"]),
+  button:where([aria-pressed="true"]) {
+    @include mixin.nl-button--default-pressed;
+  }
+
+  input[type="button" i]:where(:hover:not([aria-disabled="true"], :disabled)),
+  input[type="reset" i]:where(:hover:not([aria-disabled="true"], :disabled)),
+  input[type="submit" i]:where(:hover:not([aria-disabled="true"], :disabled)),
+  button:where(:hover:not([aria-disabled="true"], :disabled)) {
+    @include mixin.nl-button--default-hover;
+  }
+
+  input[type="button" i]:where(:active:not([aria-disabled="true"], :disabled)),
+  input[type="reset" i]:where(:active:not([aria-disabled="true"], :disabled)),
+  input[type="submit" i]:where(:active:not([aria-disabled="true"], :disabled)),
+  button:where(:active:not([aria-disabled="true"], :disabled)) {
+    @include mixin.nl-button--default-active;
+  }
+
+  /* override the `:focus` selector above */
+  input[type="button" i]:focus-visible,
+  input[type="reset" i]:focus-visible,
+  input[type="submit" i]:focus-visible,
+  button:focus-visible {
+    @include mixin.nl-button--focus-visible;
```

Comment:
Nu gaat het misschien goed, maar hebben we een accepatiecriterium dar de disabled focus state als enige visuele effect moet hebben dat er een focus ring is, en geen background color?

Misschien hier ook even een comment plaatsen, to that effect